game.py

```diff
@@ -25,16 +25,36 @@
 # Load pyglet graphic window from the frontend module.
 window = init_window(state)
 
-
 @window.event
 def on_draw():
     """
-    Clears the graphic window and draw the game state (board and robots).
+    Draw the game state (board and robots) and react to user's resizing of window by scaling the board.
     """
+    
+    # find out maximal screen size of user
+    platform = pyglet.window.get_platform()
+    display = platform.get_default_display()
+    screen = display.get_default_screen()
+    screen_width = screen.width
+    screen_height = screen.height
```

Comment:
Jeste me zarazi ty to screen_width a screen_height. K cemu jsou? Dal se nepouzivaji.

Reply:
@anezkamll, prosím o review.